docs/config/README.md

```diff
@@ -245,13 +245,14 @@ it would have been `nixpkgs/pkgs`.
 
 ### Options
 
-| Variable                    | Default       | Description                                                                      |
-| --------------------------- | ------------- | -------------------------------------------------------------------------------- |
-| `truncation_length`         | `3`           | The number of parent folders that the current directory should be truncated to.  |
-| `truncate_to_repo`          | `true`        | Whether or not to truncate to the root of the git repo that you're currently in. |
-| `fish_style_pwd_dir_length` | `0`           | The number of characters to use when applying fish shell pwd path logic.         |
-| `style`                     | `"bold cyan"` | The style for the module.                                                        |
-| `disabled`                  | `false`       | Disables the `directory` module.                                                 |
+| Variable                    | Default       | Description                                                                              |
+| --------------------------- | ------------- | ---------------------------------------------------------------------------------------- |
+| `truncation_length`         | `3`           | The number of parent folders that the current directory should be truncated to.          |
+| `truncate_to_repo`          | `true`        | Whether or not to truncate to the root of the git repo that you're currently in.         |
+| `fish_style_pwd_dir_length` | `0`           | The number of characters to use when applying fish shell pwd path logic.                 |
+| `use_logical_path`          | `true`        | Displays the logical path provided by the shell (`PWD`) instead of the path from the OS. |
```

Comment:
I think the majority of users wouldn't be able to tell the difference, let alone knowing that they want to change this configuration option.

I really like what was being done here: https://github.com/starship/starship/pull/316/files/29f07ff0956e6a086baa3eb5ab067acb36bcb02e#diff-170a3f2c0a87c4f6c25b4d944865f1c8R121-R131
What are your thoughts around adding an "Advanced configuration" table that is hidden within a spoiler for configuration options that wouldn't be immediately visible to the average user?

Reply:
I think that makes sense.